extensions/answer_summarizers/models.py

```diff
@@ -74,6 +74,14 @@ def _get_top_answers_by_frequency(answers, limit=None):
     limit answers, and returns an AnswerFrequencyList.
 
     This method is run from within the context of a MapReduce job.
+
+    Args:
+        answers: iterable of *. The collection of answers to be tallied.
```

Comment:
Just to check -- we can't say list(*)? More deterministic is better, if possible, to prevent confusion between strings/lists etc.

Reply:
Ah well I don't want to be too picky here since a lot of my code is already passing generator expressions for efficiency.